lib/test/requirements_resolver.rb

```diff
@@ -269,6 +269,9 @@ def validate_task_config_groups!
       !file_changed?('app/javascript/types/index.ts') &&
         !diff_mentions?('intersection')
     end,
+    Test::Tasks::StartPercy => proc do
+      ENV['PERCY_TOKEN'].blank?
+    end,
```

Comment:
(Mostly unrelated change that just made it a little easier/smoother for me to test this PR on my local machine.)